lib/razorpay/invoice.rb

```diff
@@ -20,5 +20,43 @@ def self.fetch(id)
     def self.all(options = {})
       request.all options
     end
+
+    def self.update(id, data = {})
+      request.patch id, data
+    end
+
+    def self.issue(id)
+      request.post "#{id}/issue"
+    end
+
+    def self.cancel(id)
+      request.post "#{id}/cancel"
+    end
+
+    def update(data = {})
```

Comment:
Can we call this `edit` instead? In keeping with the equivalent route for customers.

Reply:
Changed name to `edit` as suggested